packages/storybook/stories/MessageThread/MessageThread.stories.tsx

```diff
@@ -384,7 +389,8 @@ const Docs: () => JSX.Element = () => {
         <DetailedBetaBanner />
         <Description>
           The following example shows how to enable rich text editor for message editing by providing the
-          `richTextEditor`` property.
+          `richTextEditor` property. Rich text editor does not support mentioning users at the moment. By set
```

Comment:
Not sure if it should be `By setting`

Reply:
Ya i think this should be 'setting' as well